library/core/src/fmt/mod.rs

```diff
@@ -431,7 +431,7 @@ impl<'a> Arguments<'a> {
     /// ```
     #[unstable(feature = "fmt_as_str", issue = "74442")]
     #[inline]
-    pub fn as_str(&self) -> Option<&'static str> {
+    pub const fn as_str(&self) -> Option<&'static str> {
```

Comment:
Same here, I cannot see a `const` user. I think this change doesn't do anything.